recipes/minimize_access.rb

```diff
@@ -65,3 +65,18 @@
     group 'root'
   end
 end
+
+# Test kitchen for Ubuntu and Amazon Linux 2 is expecting cron files and folders to not be readable or writable by groups or others
+file '/etc/crontab' do
```

Comment:
I am unclear how this is related to run audit with amazon linux?

Reply:
It is not. However, in my first attemps, tests were failing because crontab permissions weren't set right.
See the failures here: https://travis-ci.org/github/dev-sec/chef-os-hardening/jobs/762137529
At worst, I can remove and create another PR

Reply:
Moved it to a different PR, see #273